css/interface.css

```diff
@@ -18,6 +18,16 @@ body {
   align-items: center;
 }
 
+.desc.sorted::after {
+  font: normal normal normal 14px/1 FontAwesome;
```

Comment:
@tonytlwu you good with this?

Reply:
I don't know yet. There are lots of errors with the screencast video. I'll share my thoughts in Jira.